src/main/java/de/uni_tuebingen/ub/nppm/model/Einzelbeleg.java

```diff
@@ -952,4 +957,65 @@ public Integer getKritikId() {
     public void setKritikId(Integer kritikId) {
         this.kritikId = kritikId;
     }
+
+    //Utility Function
+    public static List<EinzelbelegMghLemma_MM> getLemmaByEinzelbelegId(int einzelbelegId) throws Exception {
+        try ( Session session = getSession()) {
```

Comment:
Functions in model classes should never create their own queries, they should always use only class properties and methods to get ORM-related information